timetracker/entries/views.py

```diff
@@ -8,3 +8,26 @@ def entries(request):
     return render(request, 'entries.html', {
         'entry_list': entry_list,
     })
+
+def projects(request):
+    project_list = Project.objects.all()
+    return render(request, 'projects.html', {
+        'project_list': project_list,
+    })
+
+def clients(request):
+    client_list = Client.objects.all()
+    return render(request, 'clients.html', {
+        'client_list': client_list,
+    })
+
+def client_summary(request, id):
```

Comment:
Try to avoid using Python keywords for variable names.  It's not an error, it's just "sometimes hazardous".